modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase

```diff
@@ -35,6 +35,20 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172458. User can access Deal Registration Listing through Quick Links"
+	@priority = "4"
+	test CanAccessDealListingViaQuickLinks {
+		task ("When I click on 'MDF' button, on Quick Links section") {
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "deal-registrations",
+				buttonText = "Deal Registration");
+		}
+
+		task ("Then I'm redirected to the 'MDF Request Listing' page") {
+			PRMPages.assertPageTitle(pageTitle = "Partner Deal Registration");
```

Comment:
Hi @juliosilveira77, could you please rename this task to: "Then I'm redirected to the 'Deal Registration Listing' page"?